### PR TITLE
allow aerospike backend to specify set to use but still default to ol…

### PR DESCRIPTION
**Pull request by grossjo**
…d constant, uuid, if not used

### CODE REVIEW

File: backends/aerospike_test.go [hunk 1]
```diff
@@ -89,8 +89,8 @@ func (c *goodAerospikeClient) Put(policy *as.WritePolicy, aeKey *as.Key, binMap
 	return as_types.NewAerospikeError(as_types.KEY_MISMATCH)
 }
 
-func (c *goodAerospikeClient) NewUuidKey(namespace string, key string) (*as.Key, error) {
-	return as.NewKey(namespace, setName, key)
+func (c *goodAerospikeClient) NewUuidKey(namespace string, set string, key string) (*as.Key, error) {
+	return as.NewKey(namespace, set, key)
 }
 
 func TestNewAerospikeBackend(t *testing.T) {
```
Comment (hhhjort) [inline backends/aerospike_test.go:96]: I notice that our tests are not updated, and we are thus passing a config with an empty `Set` value. Should we update the tests to have a non-null value, and perhaps modify the code to throw an error if the value is missing?

Comment (grossjo) [reply backends/aerospike_test.go:96]: Good point, I'll add some extra validation around this.
Although this should never be empty in actual run, as it defaults on missing to the old hardcoded constant.

Comment (hhhjort) [reply backends/aerospike_test.go:96]: Are you planning on adding this validation soon?

Comment (grossjo) [reply backends/aerospike_test.go:96]: I was waiting for this PR https://github.com/prebid/prebid-cache/pull/81 to be merged first.
As these two will have conflicts.

I'll close this for now and reopen it in the future.

**Comment by grossjo** [comment]
Will open again in the future.

**Pull request closed without merging**